contxt/utils/__init__.py

```diff
@@ -62,3 +64,15 @@ def __get__(self, obj, objtype=None):
 
     def __repr__(self):
         return f"<{self.__class__.__name__} func={self.func}>"
+
+
+def get_http_user_agent() -> str:
+    """Returns a user-agent string based on `__version__`"""
+    try:
+        [user, version] = __version__.split(",")
```

Comment:
is this right? `__version__` should just be the version string like `5.0.1`

Reply:
@branchvincent whoops, that's right! 

mistook `@click.version_option` as being the actual output of `--version`

thanks!